spec.md

```diff
@@ -283,6 +293,69 @@ intmodule MyIntrinsicModule_xhello_y64 :
   parameter y = 42
 ```
 
+The types of intrinsic module parameters may only be literal integers or
+string literals.
+
+A parameter that is a string-encoded integer literal is treated as a string
+literal.
```

Comment:
This is why I don't like this. `Integers` should be `42`, not `"hFFFF"`. The fact that this is now a corner case here is just bad. It should accept strings. The ONLY place that it makes sense to pass in a hex number is when a user cares about a hardware literal value, hence, it should only be special cased in the grammar there.

Reply:
by "it should accept strings" do you mean "it should not accept strings"?

Reply:
I think my point is that "string-encoded-integers" should be treated in the grammar as a 'subtype' of String. But, I'm actually in favor of the proposal of removing them altogether and just providing more syntaxes for declaring integers from different bases. E.g. h-FFFF or whatever.

Reply:
Alternately the user should never be reading firrtl, so it doesn't matter if firrtl doesn't support hex, that has nothing to do with whether they can write hex numbers in chisel.